### PR TITLE
[nginx-ingress-controller] Add annotation to add CORS support

### PR DESCRIPTION
**Pull request by aledbf**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/contrib/1773)

### CODE REVIEW

File: ingress/controllers/nginx/nginx/cors/main.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	if ok {
+		if b, err := strconv.ParseBool(val); err == nil {
+			return b
+		}
+	}
+	return false
+}
+
+// ParseAnnotations parses the annotations contained in the ingress
+// rule used to indicate if the upstream servers should use SSL
```
Comment (bprashanth) [inline ingress/controllers/nginx/nginx/cors/main.go]: please fix comment

**Review by bprashanth — COMMENTED** [review]
LGTM after the nit fix

File: ingress/controllers/nginx/nginx/cors/main.go (hunk 1 above)
Comment (aledbf) [inline ingress/controllers/nginx/nginx/cors/main.go]: done

**Comment by aledbf** [comment]
I signed it

**Comment by vinay-g** [comment]
@aledbf - can you give an example on how to enable cors in ingress, I added annotation `ingress.kubernetes.io/enable-cors: "true"` but could not get this work. I tried this using `nginx-ingress-controller:0.8.3`

**Comment by aledbf** [comment]
@vinay-g this feature is not available in 0.8.3. Next release

**Pull request merged**